docs/content/en/_index.md

```diff
@@ -22,33 +22,46 @@ cascade:
 
 HEIR is a compiler toolchain for
 [fully homomorphic encryption](https://en.wikipedia.org/wiki/Homomorphic_encryption)
-(FHE). We aim to standardize a set of intermediate representations related to
-FHE, which compiler engineers, hardware designers, and cryptography researchers
-can build upon to accelerate the research and development of production-strength
-privacy-first software systems.
+(FHE). We aim to be the industry-standard compiler for FHE. Application
+developers, Compiler engineers, hardware designers, and cryptography researchers
+can build upon HEIR to accelerate the research and development of
+production-strength privacy-first software systems.
 
-HEIR is built in the [MLIR](https://mlir.llvm.org/) framework.
+## Why HEIR?
 
-For an overview of the project's goals, see
-[our talk at FHE.org](https://www.youtube.com/watch?v=kqDFdKUTNA4).
+For application developers, HEIR aims to provide a simple entrypoint to start
+working with FHE. Write a program in Python, annotate the types to mark which
+are secret, and HEIR will compile the rest.
+
+For hardware designers, HEIR provides multiple layers of abstraction at which to
+integrate code generation. This allows HEIR to support code generation for
+hardware accelerators that implement high-level FHE operations like bootstrap,
+as well as accelerators that operate at lower-level polynomial arithmetic.
 
-To see the dialects and possible flows, take a look at the diagram below: {{%
-figure src="/images/dialects.svg" link="/images/dialects.svg" %}}
+For cryptography researchers, HEIR provides a convenient platform for research.
+HEIR providines the compiler infrastructure and implements standard
```

Comment:
```suggestion
HEIR provides the compiler infrastructure and implements standard
```

Reply:
I guess the pre-commit checks don't do markdown, or why did this not trigger codespell?